source/util.js

```diff
@@ -63,3 +65,22 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+function splitFileNameList(stdout) {
+	if (typeof stdout !== 'string') {
+		throw new TypeError('expected parameter stdout of type string');
+	}
+
+	const rows = stdout.trim().split('\n');
+	const result = [];
+	for (let i = 0; i < rows.length; i++) {
+		result.push(rows[i].substring(0, rows[i].indexOf('|')).trim());
+	}
+
+	return result;
```

Comment:
Use something like:

```js
return stdout.trim().split('\n').map(row => rows.replace(/\|+\|/, '').trim())
```

Reply:
I used this function, you suggested.
The ```git diff```-command returns the filename followed by ```| +++---``` for added or deleted rows.
So the regex looks a bit different to yours.